tests/ui/stats/hir-stats.stderr

```diff
@@ -39,37 +39,37 @@ ast-stats-1 - Match                     72 ( 1.1%)             1
 ast-stats-1 - Struct                    72 ( 1.1%)             1
 ast-stats-1 - Lit                      144 ( 2.2%)             2
 ast-stats-1 - Block                    216 ( 3.3%)             3
-ast-stats-1 PathSegment              720 (10.9%)            30            24
```

Comment:
`PathSegment` hasn't changed in size. Something about the visitor must have changed, but I can't see what. Do you know?

Reply:
This comment is relevant, it describes how the AST node measurements aren't perfect:

https://github.com/rust-lang/rust/blob/4bc39f028d14c24b04dd17dc425432c6ec354536/compiler/rustc_passes/src/hir_stats.rs#L47-L63

Reply:
Paths in attributes are now visited.